packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -73,14 +73,52 @@ class ExpansionTile extends StatefulWidget {
     this.collapsedShape,
     this.clipBehavior,
     this.controlAffinity,
-  }) : assert(initiallyExpanded != null),
+  }) : child = null,
+  assert(initiallyExpanded != null),
        assert(maintainState != null),
        assert(
        expandedCrossAxisAlignment != CrossAxisAlignment.baseline,
        'CrossAxisAlignment.baseline is not supported since the expanded children '
            'are aligned in a column, not a row. Try to use another constant.',
        );
 
+  /// a constructor for the [ExpansionTile] that allows a single [child] instead of a list of [children].
```

Comment:
This seems a bit confusing, why do we need to have both `children` and `child`?  Can children not have a length of 1?

Reply:
Thanks for the review @piinks, This approach was based on this comment https://github.com/flutter/flutter/issues/39188#issuecomment-525360630

Using a child will give freedom to add any type of widgets including scrollable or flexible widgets (e.g expanded within a column).